assets/src/state.ts

```diff
@@ -0,0 +1,75 @@
+import { Dispatch as ReactDispatch } from "react"
+import { Route, RouteId } from "./skate.d"
+
+export interface State {
+  routes: null | Route[]
```

Comment:
This `null | Route[]` is repeated in routePicker - maybe worth making a named type for it?

Reply:
I tried out defining `export type LoadingRoutes = null | Route[]` in `state.ts`. The indirection and verbosity ended up being worse than the duplication.